docs/source/userguide/create/index.rst

```diff
@@ -124,102 +94,6 @@ Example:
-
-Options:
-::
-
-    usage: autosubmit testcase [-h] [-y COPY | -min] -d DESCRIPTION [-c CHUNKS]
-                               [-m MEMBER] [-s STARDATE] -H HPC [-repo GIT_REPO]
-                               [-b GIT_BRANCH] [-conf GIT_AS_CONF] [-local]
-    
-    create test case experiment
-    
-    optional arguments:
-      -h, --help            show this help message and exit
-      -y COPY, --copy COPY  makes a copy of the specified experiment
-      -min, --minimal_configuration
-                            creates a new experiment with minimal configuration,
-                            usually combined with -repo
-      -d DESCRIPTION, --description DESCRIPTION
-                            description of the test case
-      -c CHUNKS, --chunks CHUNKS
-                            chunks to run
-      -m MEMBER, --member MEMBER
-                            member to run
-      -s STARDATE, --stardate STARDATE
-                            stardate to run
-      -H HPC, --HPC HPC     HPC to run experiment on it
-      -repo GIT_REPO, --git_repo GIT_REPO
-                            sets a git repository for the experiment
-      -b GIT_BRANCH, --git_branch GIT_BRANCH
-                            sets a git branch for the experiment
-      -conf GIT_AS_CONF, --git_as_conf GIT_AS_CONF
-                            sets the git path to as_conf
-      -local, --use_local_minimal
-                            uses local minimal file instead of git
-
-Example:
-::
-
-    autosubmit testcase -d "TEST CASE cca-intel auto-ecearth3 layer 0: T511L91-ORCA025L75-LIM3 (cold restart) (a092-a09n)" -H cca-intel -b 3.2.0b_develop -y a09n
-
-Test the experiment
--------------------
-
-This method is to conduct a test for a given experiment. It creates a new experiment for a given experiment with a
-given number of chunks with a random start date and a random member to be run on a random HPC.
-
-To test the experiment, use the command:
-::
-
-    autosubmit test CHUNKS EXPID
-
-*EXPID* is the experiment identifier.
-*CHUNKS* is the number of chunks to run in the test.
-
-
-
-Options:
-::
-
-    usage: autosubmit test [-h] -c CHUNKS [-m MEMBER] [-s STARDATE] [-H HPC] [-b BRANCH] [-v] EXPID
-
-     test experiment
-
-     positional arguments:
-        EXPID                 experiment identifier
-
-     options:
-         -h, --help            show this help message and exit
-         -c CHUNKS, --chunks CHUNKS
-                               chunks to run
-         -m MEMBER, --member MEMBER
-                               member to run
-         -s STARDATE, --stardate STARDATE
-                               stardate to run
-         -H HPC, --HPC HPC     HPC to run experiment on it
-         -b BRANCH, --branch BRANCH
-                               branch of git to run (or revision from subversion)
-         -v, --update_version  Update experiment version
-
-Example:
-::
-
-    autosubmit test -c 1 -s 19801101 -m fc0 -H ithaca -b develop cxxx
-
```

Comment:
GitHub UI is showing as if this command, but also all the text above about test cases was deleted. Is that really the case, and was it intentional?

Reply:
Ah, got it. So let's remove that function too. Can you create a pull request for that?

Reply:
Scratch that, I have some spare time now. Let me take care of that! :rocket: